### PR TITLE
Doc links for slo, control-api, jwt-security

### LINKED ISSUE #554: Add docs links for control-api, jwt-security, slo to dashboard
all 3 docs links are missing in the release-2.2 catalog json configs

### PR DESCRIPTION
**Pull request by bcmendoza**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Closes #554

2. What **changes to custom.yaml** are required?

None

3. Have you documented any additional setup steps or configurations options?

None

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Visit the dashboard and click the documentation link icons for SLO, JWT Security, and Control API. The browser should navigate to each documentation page at `/services/{serviceName}/latest/`.

### CODE REVIEW

File: sense/Chart.yaml [hunk 1]
```diff
@@ -15,15 +15,11 @@ keywords:
 dependencies:
   - name: catalog
     repository: file://./catalog
-    version: '2.2.0'
+    version: '2.2.1'
 
   - name: dashboard
     repository: file://./dashboard
-<<<<<<< HEAD
```
Comment (moosecanswim) [inline sense/Chart.yaml:22]: @bcmendoza ya got a merge conflict here but i think this makes sense.  increment the child chart and the parent chart.

@chrisbsmith could you confirm if this enough for a patch version increment

Comment (chrisbsmith) [reply sense/Chart.yaml:22]: Since the change is isolated to just the configs, I don't think this is necessary for a version bump.

Comment (bcmendoza) [reply sense/Chart.yaml:22]: @moosecanswim @chrisbsmith Got it! Reverted back to 2.2 for everything. Can I get your reviews as well?

**Comment by moosecanswim** [comment]
LGTM

**Pull request merged**